tests/functional/samples/test_samples_linux.py

```diff
@@ -109,6 +109,33 @@ def test_hello_classification_cpp_hddl(self, is_distribution, is_image_os, teste
              ], self.test_hello_classification_cpp_hddl.__name__, **kwargs,
         )
 
+    @pytest.mark.parametrize('is_distribution', ['dev', 'proprietary'], indirect=True)
+    @pytest.mark.parametrize('is_image_os', ['ubuntu18', 'ubuntu20'], indirect=True)
+    @pytest.mark.parametrize('is_not_product_version', ['2020.3'], indirect=True)
+    @pytest.mark.xfail(reason='invalid model')
```

Comment:
don't use xfail for really negative tests, you need catch raised exception FailedTest with exit code and check caplog with appropriate error text.